services/wallet/api.go

```diff
@@ -574,7 +581,28 @@ func (api *API) SendTransactionWithSignature(ctx context.Context, chainID uint64
 
 func (api *API) CreateMultiTransaction(ctx context.Context, multiTransactionCommand *transfer.MultiTransactionCommand, data []*bridge.TransactionBridge, password string) (*transfer.MultiTransactionCommandResult, error) {
 	log.Debug("[WalletAPI:: CreateMultiTransaction] create multi transaction")
-	return api.s.transactionManager.CreateMultiTransactionFromCommand(ctx, multiTransactionCommand, data, api.router.bridges, password)
+
+	selectedAccount, err := api.getVerifiedWalletAccount(multiTransactionCommand.FromAddress.Hex(), password)
```

Comment:
@IvanBelyakoff this will return an error if the related keypair is migrated to a keycard.